packages/pluggableWidgets/radio-buttons-native/README.md

```diff
@@ -1,24 +1,22 @@
 ## RadioButtons
-[My widget description]
 
-## Features
-[feature highlights]
-
-## Usage
-[step by step instructions]
+-   The radio buttons widget is a collection of radio buttons representing a set of related options. Only one radio
+    button is allowed to be selected in a given collection.
 
-## Demo project
-[link to sandbox]
+## Features
 
-## Issues, suggestions and feature requests
-[link to GitHub issues]
+-   Buttons orientation is configurable - Vertical or Horizontal.
+-   Different style for read-only collections.
+-   color controls for buttons and label.
+-   Dynamic styles can be added.
 
-## Development and contribution
+## Usage
 
-1. Install NPM package dependencies by using: `npm install`. If you use NPM v7.x.x, which can be checked by executing `npm -v`, execute: `npm install --legacy-peer-deps`.
-1. Run `npm start` to watch for code changes. On every change:
-    - the widget will be bundled;
-    - the bundle will be included in a `dist` folder in the root directory of the project;
-    - the bundle will be included in the `deployment` and `widgets` folder of the Mendix test project.
+1. Drag the Radio buttons widget into a native page.
+2. Select a data source.
+3. Select the orientation - vertical is default.
+4. Optionally Add label for the collection.
+5. Run the Mendix application to check out the radio buttons.
 
-[specify contribution]
+## Custom styling
+The look and feel of the chart is fully customizable. Check out the <a href="https://docs.mendix.com/refguide/native-styling-refguide#6-6-reference-selector">Native Mobile Styling documentation</a> for a full overview of the available style customizations.
```

Comment:
This href points to the reference selector, not the new widget?

Reply:
yes, because it is not merged yet. What should be done in this case?

Reply:
In the docs.mendix.com PR you know which "section" the new content will be, so you can use that href, so long as you don't run into a merge conflict on docs.mendix.com :)

Reply:
Can you please check now?

Reply:
Looking at your code changes to docs.mendix.com, you introduced section 6.7.